quarkchain/cluster/grpc_client.py

```diff
@@ -68,5 +68,7 @@ def get_unconfirmed_header(self) -> Optional[List[Any]]:
         except grpc.RpcError:
             Logger.log_exception()
             return None
-
+        Logger.info(
+            "Length of unconfirmed headers: {}".format(len(response.header_list))
```

Comment:
also log number of minor block headers in the root block when calling `AddRootBlock`?

Reply:
Ok~